paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -207,21 +208,17 @@ internal abstract class BaseSheetViewModel(
         }
     }.distinctUntilChanged()
 
-    val ctaEnabled = MediatorLiveData<Boolean>().apply {
```

Comment:
I vote we do renames separate from behavior changes in the future. Thoughts?

Reply:
It’s a small enough change in this case, where `ctaEnabled` was only used in two places. I generally agree with you though.